migrations/20220622220001-add-custom-thank-you-email-message.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Collectives', 'customThankYouMessage', {
+      type: Sequelize.TEXT,
+    });
+  },
```

Comment:
Can you just put this in the Collective settings for now?

Reply:
We are now storing this in settings. 👍🏽